### PR TITLE
9.0 Updates

### PR DESCRIPTION
**Pull request by jkelroy**
- Update workspaces page to include local workspaces
- Update preference settings to include new options

### CODE REVIEW

**Comment by jkelroy** [comment]
Should be good to go. As a note, if you are viewing this locally, there are some broken images as I will be using some images that Jonathan provided and will get merged in on his 9-0-release-notes branch.

File: gitkraken-client/workspaces.md [hunk 1]
```diff
@@ -9,136 +9,108 @@ taxonomy:
...
-
-## Workspace type
-
-Workspaces are available as either a _Personal_ or _Team_ type.
-
-**Personal Workspaces** are only accessible by you on this machine.
+## Access workspaces
 
-**Team Workspaces** will be available for the selected teams within your organization. This helps ensure that everyone is up-to-date on the same set of repositories by offering [multi-repository actions](/gitkraken-client/workspaces/#multi-repository-actions) and the ability to [work with all pull requests](/working-with-repositories/workspaces/#pull-requests) from these repositories. 
+To access Workspaces, select the Workspace tab in the top left or use the command palette (`command/ctrl + shift + P`) and search "Open Workspaces".
```
Comment (jjsilva4) [inline gitkraken-client/workspaces.md:16]: Should be Command Palette

File: gitkraken-client/workspaces.md [hunk 2]
```diff
@@ -9,136 +9,108 @@ taxonomy:
...
-
-## Edit a workspace
-
-Edit a workspace by selecting the ellipsis <i class="fas fa-ellipsis-v"></i> icon by the Workspace name.
-
-<img src="/wp-content/uploads/edit-a-workspace.png" srcset="/wp-content/uploads/edit-a-workspace@2x.png" class="img-bordered img-responsive center">
+***
 
-## Multi-repository actions
+## Cloud workspaces
```
Comment (jjsilva4) [inline gitkraken-client/workspaces.md:22]: Should Cloud Workspaces

File: gitkraken-client/workspaces.md [hunk 3]
```diff
@@ -9,136 +9,108 @@ taxonomy:
...
-
-<img src="/wp-content/uploads/repos-status.png" srcset="/wp-content/uploads/repos-status@2x.png" class="img-bordered img-responsive center">
-
-## Workspace breadcrumb in toolbar
-
-The option to remove the Workspace breadcrumb in the toolbar can be toggled under <kbd> Preferences > UI Customization > _Show Workspace breadcrumb in toolbar_</kbd>.
+<div class='callout callout--warning'>
+    <p>
+        <strong>Note:</strong>
+            To create a Cloud Workspace, the integration for the desired service must be connected under <kbd>Preferences > Integrations</kbd>.
```
Comment (jjsilva4) [inline gitkraken-client/workspaces.md:35]: Switch to active voice.

File: gitkraken-client/workspaces.md [hunk 4]
```diff
@@ -152,17 +124,82 @@ The Team Pull Requests section has the following columns:
...
+
+<img src="/wp-content/uploads/repos-status.png" srcset="/wp-content/uploads/repos-status@2x.png" class="img-bordered img-responsive center">
+
+***
+
+## Workspace breadcrumb in toolbar
+
+The option to remove the Workspace breadcrumb in the toolbar can be toggled under <kbd> Preferences > UI Customization > _Show Workspace breadcrumb in toolbar_</kbd>.
+
+<img src="/wp-content/uploads/breadcrumb-setting.png" srcset="/wp-content/uploads/breadcrumb-setting@2x.png" class="img-bordered img-responsive center">
```
Comment (jjsilva4) [inline gitkraken-client/workspaces.md:205]: No comment about this code, but I suspect I'll have more feedback when we push this production tomorrow morning.

**Review by jjsilva4 — CHANGES_REQUESTED** [review]

**Pull request merged**